README.md

```diff
@@ -20,7 +20,6 @@ TOKEN_ISSUER = ''
 TOKEN_AUDIENCE = ''
 
 # Specifies the allowed validity window to accommodate clock skew
-# between the given issue time and expiration time (in minutes)
-# that the ID token is within
-TOKEN_LEEWAY = ''
+# between the given expiration time of the ID token
+TOKEN_LEEWAY = seconds
```

Comment:
This is called UW_TOKEN_LEEWAY in id_token.py.

Reply:
Ah, thanks!